File: 백준/Silver/1541. 잃어버린 괄호/잃어버린 괄호.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim();
const splitedStr = input.split('-');
let result = splitedStr[0].split('+').map(Number).reduce((acc, cur) => acc + cur, 0);

for(let i=1; i<splitedStr.length; i++) {
    let sum = splitedStr[i].split('+').map(Number).reduce((acc,cur) => acc+cur,0);
    
    result -= sum;
}

console.log(result);

File: 백준/Silver/1541. 잃어버린 괄호/잃어버린 괄호.py
import sys
input = sys.stdin.readline

expression = input().strip()
parts = expression.split('-')

total = 0
for i, part in enumerate(parts):
    if i == 0:
        total += sum(map(int, part.split('+')))
    else:
        total -= sum(map(int, part.split('+')))

# 결과 출력
print(total)